extensions/optaplanner/deployment/src/main/java/io/quarkus/optaplanner/deployment/SolverBuildTimeConfig.java

```diff
@@ -15,6 +15,8 @@
 @ConfigGroup
 public class SolverBuildTimeConfig {
 
+    public static final String DEFAULT_SCORE_DRL_URL = "scoreRules.drl";
```

Comment:
Should change to "constraint.drl" per latest discussion.

Reply:
@ge0ffrey so this PR is not ready to be merged? Just to be sure.

Reply:
FYI the original file is plural, so I believe `constraints.drl` would make more sense :wink:

Reply:
Updating PR now to resolve those 2 conversations, will mention when it's ok to merge.